site/docs/v1/tech/tutorials/register-user-login-api.adoc

```diff
@@ -1,26 +1,48 @@
 ---
 layout: doc
-title: Register a User & Login
-description: Learn how to register a User and log them in via the FusionAuth API
+title: Register a User and Login
+description: Learn how to register a User and log them in
 navcategory: getting-started
 
 ---
 
 == Overview
 
-This tutorial guides you through the basics of registering users and logging them into Applications using the FusionAuth APIs.
+This tutorial guides you through the basics of registering users and logging them into Applications using the FusionAuth APIs as well as some alternatives.
 
 
 == Register a User
 
 In order to register a User, you must have first created an Application.  A tutorial for creating an application is provided in the link:/docs/v1/tech/core-concepts/applications[Application overview]. Once the Application has been created, you are ready to call the API to register a User.
 
+
 There are two APIs that can be used to create the User and then create a Registration for that User in the Application you create. In most cases you will want to create the User and register them in a single step. This can be accomplished by calling the link:/docs/v1/tech/apis/registrations#create-a-user-and-registration-combined[/api/user/register (combined) API].
 
 However, you can also create the User and then register them for the Application in separate API calls. This method would make use of the link:/docs/v1/tech/apis/users#create-a-user[/api/user API] followed by a call to the link:/docs/v1/tech/apis/registrations#create-a-user-registration-for-an-existing-user[/api/user/register (for an existing user) API]. We recommend using the single API call, but in some cases, calling the APIs separately is preferred.
 
+You can also allow accounts to be created with link:/docs/v1/tech/guides/basic-registration-forms[self-service registration]. This uses pages hosted by FusionAuth.
+
+image::guides/basic-registration-forms/register-new-user.png[Example of built out basic registration form.,width=1200, role=bottom-cropped]
+
+== Log in a User
+
+Once you have create a User and registered them for an Application, you can authenticate them by calling the link:/docs/v1/tech/apis/login[Login API]. That will return a JWT and other User information as documented.
+
+A user can also log in via the link:/docs/v1/tech/core-concepts/integration-points/#hosted-login-pages[hosted login pages]. These use the Authorization Code grant and will redirect the user to FusionAuth.
+
+image::tutorials/login-page.png[Example of login form.,width=1200, role=bottom-cropped]
+
+== Should I Use the APIs or The Hosted Pages
+
+In general the hosted login pages are recommended. They are link:/docs/v1/tech/themes/[customizable and localizable with themes] and when used, FusionAuth is the only server side system to see sensitive user information like credentials.
```

Comment:
FusionAuth is the only server-side system to see sensitive user information like credentials (in other words, you are delegating authentication completely to FusionAuth).